azure-keyvault/src/main/java/com/microsoft/azure/keyvault/models/StorageAccountItem.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Code generated by Microsoft (R) AutoRest Code Generator 1.2.2.0
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+package com.microsoft.azure.keyvault.models;
+
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The storage account item containing storage account metadata.
+ */
+public class StorageAccountItem {
```

Comment:
This is a model from the updated spec we should remove it from the refactoring, and add back in a subsequent update.